source/renderer/app/api/etc/sendEtcBugReport.js

```diff
@@ -16,35 +16,22 @@ export type SendEtcBugReportRequestParams = {
 
 export const sendEtcBugReport = (
   { requestFormData, application }: SendEtcBugReportRequestParams
-): Promise<{}> => {
+): Promise<void> => {
```

Comment:
This basically circumvents typing (except of the fact that we are returning a promise) … shouldn't we define the response type here?

Reply:
Because we are actually not returning anything!
I am not sure if this is right - this was required for both lint and flow:type to run successfully...

Reply:
We are returning the promise that is created by the typed `request` method 😉 so the type you define here gets passed down to that request method so that flow knows that this is what we expect to come back as a result from the server side. But im not sure what the server actually returns (as we don't have type information about this yet)